test/evaluators.py

```diff
@@ -290,6 +290,42 @@ async def __call__(
                 )
         return score
 
+class ManualContentEvaluator(Evaluator):
+    """Evaluation Route for Manual Evaluation."""
+    async def __call__(
+        self,
+        task_config: dict[str, Any],
+        page: Page,
+        client: CDPSession | None = None,
+        answer: str | None = None
+    ) -> float:
+        """Pauses Execution to get manual evaluation score from user.
+
+        Parameters:
+            task_config (dict[str, Any]): The task configuration containing evaluation criteria.
+            page (Page): The Playwright page object for the current webpage.
+            client (CDPSession | None, optional): The Chrome DevTools Protocol session object. Not used in this evaluator.
+            answer (str | None, optional): Not used in this evaluator.
+
+        Returns:
+            float: A score between 0.0 and 1.0 representing the presence of required HTML content on the webpage.
+        """
+        task:str = task_config["intent"]
+        reference_answer=task_config["eval"]["reference_answers"]["manual_check"]["answer"]
+        answer_type=task_config["eval"]["reference_answers"]["manual_check"]["type"]
+        id=task_config["task_id"]
+        print("Task ID: ",id)
```

Comment:
do you think it would be good to change the print statements to logger statements? This way it is uniform?